### PR TITLE
Ports far too many haircuts and beards.

### PR DESCRIPTION
**Pull request by pireamaineach**
inb4 snowflake
Ports some hairstyles, mainly from vore and bay.

Variety is the spice of life.

Changelog
:cl:
add: Adds far too many haircuts.
add: Adds a decent amount of beards.
/:cl:

### CODE REVIEW

**Comment by daklaj** [comment]
more space for cancer snowlake players
there should be 3 hair styles: bald, short and long at 3 hair colors at max

**Comment by subject217** [comment]
Did you make all these yourself, or port them from somewhere?

**Comment by pireamaineach** [comment]
I ported some from bay and regret to say some are from Cit.

**Comment by subject217** [comment]
Okay, you need to give credit for them. At least in the PR itself.

**Comment by pireamaineach** [comment]
No idea who made any of these.
The person who added them to Cit didn't credit anyone, and the bay ones are at a certain point just too old.

**Comment by pireamaineach** [comment]
I'll at least say they're ported.

**Comment by subject217** [comment]
You can just put it in the title and CL. Like "Ports some haircuts and beards from Citadel and Baystation".

**Comment by pireamaineach** [comment]
Problem is Cit didn't make these. They ported them from vore, and the vore ones don't really have consistent authors.

**Comment by subject217** [comment]
It's not a huge deal, just try to make a good faith attempt to give credit to the *community* that they are from.

**Comment by PKPenguin321** [comment]
needs more descriptive PR body, ESPECIALLY pictures
after a cursory glance though, 👎 because of this random undocumented change:
image
and also this which is actually cursed:
image

**Comment by pireamaineach** [comment]
Oh yeah, I forgot about that one.

Sorry for making broken man not look like absolute shit.

Also I'm not a fan of the floor length hairstyles, but I had a sprite and there's not really that much justification for leaving it out.

Then again this also assumes that PKPenguin would be logical.

**Comment by PKPenguin321** [comment]
i can think of at least one reason for leaving it out

**Comment by PKPenguin321** [comment]
such as it being floor length braid but worse

**Comment by pireamaineach** [comment]
Makes no sense, doesn't really matter.

I'm not here to only add things that I like, or you like. It'll barely see use anyway. That is, unless you whinge about it. Only if you whinge about it constantly will it see much use.

**Comment by actioninja** [comment]
Broken man is intended to look like Brad from Lisa.

**Comment by Militaires** [comment]
Broken man's eyebrow shadow is deliberate.

**Comment by Militaires** [comment]
also when are we gonna blackscale our hair like lifeweb so black hair isnt just fucking pitch black with no shading whatsoever

**Comment by pireamaineach** [comment]
Maybe the nose is a bit too much for the sprite. I'll re-add the eyes, at least.

**Comment by pireamaineach** [comment]
Nah, full revert. Not my sprite, I'm certainly not using it.

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -108,6 +108,14 @@
 	name = "Bedhead 3"
 	icon_state = "hair_bedheadv3"
 
+/datum/sprite_accessory/hair/bedheadlong
+	name = "Long Bedhead"
+	icon_state = "hair_long_bedhead"
+
+/datum/sprite_accessory/hair/bedheadfloorlength
+	name = "Floorlength Bedhead"
```
Comment (Fox-McCloud) [inline code/modules/mob/dead/new_player/sprite_accessories.dm]: oh no.

**Comment by Poojawa** [comment]
Hi

All of the ones Citadel ported are all marked. a few are homebrew afaik. 

c:

**Comment by pireamaineach** [comment]
> Hi
> 
> All of the ones Citadel ported are all marked. a few are homebrew afaik.
> 
> c:

None of them are homebrew, they're all vorestation ports.

They're only all labelled because Kevinz ran out of space in the .dm and used a shite system to circumvent it. Turns out you can copy-paste to make new lines past the limit.

**Comment by ExcessiveUseOfCobblestone** [comment]
oh wait I can just go to (bay) and (cit) to see what they look like on humans (maybe?)

**Comment by pireamaineach** [comment]
Well, didn't add all of the ones from Cit.
A fair bit of them were Taj specific. Don't even think they have Tajarans on Cit.

**Comment by ExcessiveUseOfCobblestone** [comment]
yikes lmao

**Comment by Poojawa** [comment]
we kinda do, most of the system I made is more or less a mix-and-match of parts for species.

**Comment by OverlyGeneric** [comment]
oh no, most of these are anime. oh no one of them is Jojo. what have you done?

**Comment by ArcaneMusic** [comment]
> oh no, most of these are anime. oh no one of them is Jojo. what have you done?

Oh, Contributor? Are you approaching me? Instead of closing your PR you're attempting to merge this? Even though the hairstyles are from Citadel and Vorestation, you still want to add more hairstyles to the game?

**Comment by ExcessiveUseOfCobblestone** [comment]
watashi wa being cyberbullied on the internet.

**Comment by Okand37** [comment]
More hairstyles (of good quality) is an important update to my ability to change my hair even more.

**Pull request merged**